app/services/idv/financials_validator.rb

```diff
@@ -0,0 +1,9 @@
+module Idv
+  class FinancialsValidator < VendorValidator
+    def validate
+      session_id = idv_session.resolution.session_id
+      idv_session.financials_confirmation = idv_agent.submit_financials(vendor_params, session_id)
```

Comment:
I don't think manipulating the session is the concern of the validator. I would move this task to the FinancialsStep class. See https://github.com/18F/identity-idp/pull/947